docs/build/CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+### Changelog
```

Comment:
Is there a reason to start this at a heading level of 3? All of our other markdown documents starts with a heading level of 1.

Reply:
This isn't our file. This comes from the documentation template code.

Reply:
Got it!